src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Ball {
+
+    private Color color;
```

Comment:
```suggestion
public class Ball {
    private Color color;
```
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/6871029d-a61e-40c1-a2ed-964c5e8a44b8)